.npmrc

```diff
@@ -1 +1,2 @@
 engine-strict=true
+legacy-peer-deps=true
```

Comment:
Would NPM 6 ignore this? Or does this break NPM 6? I don't mind breaking it, but if we do then we should at least call it out in the docs.

Reply:
it runs as normal with npm 6 and `npm install --legacy-peer-deps`